README.md

```diff
@@ -32,16 +35,30 @@ Initial support is offered for the following (with more to follow soon):
 - LLMs
   - An adapter is provided for Steamship's OpenAI integration (`steamship_langchain.llms.OpenAI`)
   - An adapter is provided for *caching* LLM calls, via Steamship's Key-Value store (`SteamshipCache`) 
+- Document Loaders
+  - An adapter for loading Steamship Files into Documents is provided (`steamship_langchain.document_loaders.SteamshipLoader`)
 - Tools
   - Search:
     - An adapter is provided for Steamship's SERPAPI integration (`SteamshipSERP`)
 - Memory
   - Two adapters that provide persistent conversation memory:
     - Complete Memory (`steamship_langchain.memory.ConversationBufferMemory`)
     - Windowed Memory (`steamship_langchain.memory.ConversationBufferWindowMemory`)
+- VectorStores
+  - An adapter is provided for a persistent VectorStore (`steamship_langchain.vectorstores.SteamshipVectorStore`)
+- Text Splitters
+  - A splitter for Python code, based on the AST, is provided (`steamship_langchain.python_splitter.PythonCodeSplitter`)
```

Comment:
A note here about how it's different than the one you said was in langchain?

Reply:
Good idea.